python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_multioutput_firesatellite.py

```diff
@@ -53,7 +54,8 @@
 # We choose to use a constant trend.
 
 # %%
-basis = ot.ConstantBasisFactory(m.dim).build()
+linear_basis = ot.LinearBasisFactory(m.dim).build()
+basis = ot.Basis([ot.AggregatedFunction([linear_basis.build(k)] * 3) for k in range(linear_basis.getSize())])
```

Comment:
This is not really related to this PR, but `build` is a strange name for a method that really gets the element of the basis with a given index. Shouldn't the name rather be `get`, or even better, couldn't we allow indexation, like this: ``linear_basis[k]`` ?

Reply:
The `Basis` class is independent from `kriging`. 

One of the ability of such classes is to `build on the fly` a function (polynomial for example)

linear_basis[k] yields a `TypeError` even if operator exists because it is masked in python

Thus no choice

Reply:
I don't understand: do you mean that a Basis object can build a linear combination of its elements? Where is this ability used?

Also, couldn't we unmask the [] operator in Python?

Reply:
For the second point, you might ask @jschueller There is probably a good reason for masking it

For the first point, I mean that we might generate functions on a "factory mode". If you define or embed a `LegendreFactory` into yourr `Basis`, you can ask for a `legendre` function up to a degree ... in fact no limitation as we consider infinite basis. This is why we have a `build` method here (factory mode)